app/models/post.rb

```diff
@@ -0,0 +1,19 @@
+class Post < ApplicationRecord
+  validates :title, length: { in: 1..250 }
```

Comment:
- [ ] It will be better to have validation for the presence. This is a requirement for this section. Example: ` validates :title, presence: true`